IaC/pmm.cd/JenkinsStack.yml

```diff
@@ -839,7 +839,7 @@ Resources:
               }
 
               setup_ssh_keys() {
-                KEYS_LIST="alexander.tymchuk mykola.marzhan evgeniy.patlan slava.sarzhan illia.pshonkin alex.miroshnychenko eduardo.casarero santiago.ruiz andrew.siemen kamil.babayev"
+                KEYS_LIST="alexander.tymchuk mykola.marzhan evgeniy.patlan slava.sarzhan illia.pshonkin alex.miroshnychenko eduardo.casarero santiago.ruiz andrew.siemen"
```

Comment:
where is Nikita?

Reply:
Here: https://github.com/Percona-Lab/jenkins-pipelines/pull/1041